### PR TITLE
docs updates

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by rasapala — APPROVED** [review]
LGTM

File: docs/resources.md [hunk 1]
```diff
@@ -5,9 +5,16 @@
...
+- [Face Detection with OpenVINO&trade; Model Server](./face_detection_script_example.md#example-of-face-detection-with-openvino-model-server) : This sample uses python script to run OpenVINO&trade; Face Detection Model with the Model Server.
+- [ONNX model serving](ovms_onnx_example.md)
+- [Person-Vehicle-Bike Detection](./face_detection_script_example.md#running-person-vehicle-detection-with-example-script) : Detection of Person and Vehicles using OpenVINO&trade; Model Server with python example.
+- [Age Gender Classification](./age_gender_guide.md) : Classification of Age Gender using OpenVINO&trade; Model Server.
+- [Binary Input](./binary_input.md) Sending prediction requests with images in JPEG or PNG format.
+- [DAG Scheduler pipeline with models ensemble](ensemble_scheduler.md)
+- [DAG Scheduler with OCR pipeline](east_ocr.md)
+- [DAG Scheduler pipeline with combined models](combined_model_dag.md)
+- [DAG Scheduler with vehicle detection](vehicle_detection.md)
+- [Stateful model for audio recognition](stateful_model.md)
```
Comment (mzegla) [inline docs/resources.md:17]: Those links are broken

File: docs/binary_input.md [hunk 2]
```diff
@@ -34,28 +34,6 @@ and rescale the data.
```
Comment (dkalinowski) [inline docs/binary_input.md:34]: `Support for binary inputs and outputs data in OpenVINO Model Server` title is misleading, remove `outputs` word

File: docs/performance_tuning.md [hunk 3]
```diff
@@ -40,35 +63,40 @@ In case of using CPU plugin to run the inference, it might be also beneficial to
...
 
-- To increase the throughput, a parameter `--grps_workers` is introduced which increases the number of gRPC server instances. This way the OVMS can achieve bandwidth utilization over 30Gb/s if there is sufficient network link.
+- To increase the throughput, a parameter `--grps_workers` is introduced which increases the number of gRPC server instances. In most cases the default value of `1` will be sufficient.
+  In case of particularly heavy load and many parallel connections, higher value might increase the transfer rate.
 
 - Another parameter impacting the performance is `nireq`. It defines the size of the model queue for inference execution.
 It should be at least as big as the number of assigned OpenVINO streams or expected parallel clients (grpc_wokers >= nireq).
+  
+- Parameter `file_system_poll_wait_seconds` defines how often the model server will be checking if new model version gets created in the model repository. 
+The default value is 1s which ensures prompt response to creating new model version. In some cases it might be recommended to reduce the polling frequency
```
Comment (dkalinowski) [inline docs/performance_tuning.md:82]: 1s -> 1 second

File: docs/performance_tuning.md [hunk 4]
```diff
@@ -4,24 +4,48 @@
 
 This document gives an overview of various parameters that can be configured to achieve maximum performance efficiency. 
 
-## Input data
+## Adjusting the number of Inference Engine streams in CPu and GPU target devices
```
Comment (mzegla) [inline docs/performance_tuning.md:7]:
```suggestion
## Adjusting the number of Inference Engine streams in CPU and GPU target devices
```

File: docs/performance_tuning.md [hunk 5]
```diff
@@ -4,30 +4,53 @@
...
+
+In a scenario where the number of parallel connections is close to 1, set the following parameter:
+`--plugin_config '{"CPU_THROUGHPUT_STREAMS": "1"}'`
+
+When the number of concurrent requests is higher, increase the number of streams. Make sure, however, the number of
+streams is lower then the average volume of concurrent inference operations. Otherwise the server might not be fully utilized.
+Number of streams should not exceed the number of vCPUs.
+
+For example with ~50 clients sending the requests to the server with 96 vCPU, set the number of streams to 24:
+`--plugin_config '{"CPU_THROUGHPUT_STREAMS": "24"}'`
```
[the remark this answers is not in the document]
Comment (mzegla) [reply docs/performance_tuning.md:27]: Still missing link to nireq and workers setup recommendations

**Review by mzegla — APPROVED** [review]

**Pull request merged**